addons/l10n_es_edi_tbai/data/template_invoice.xml

```diff
@@ -0,0 +1,281 @@
+                <IDVersionTBAI t-out="tbai_version"/>
+            </Cabecera>
+            <Sujetos t-if="is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_sujetos"/>
+            </Sujetos>
+            <Factura t-if="is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_factura"/>
+            </Factura>
+            <IDFactura t-if="not is_emission">
+                <t t-call="l10n_es_edi_tbai.template_invoice_sujetos"/>
+                <t t-call="l10n_es_edi_tbai.template_invoice_factura"/>
+            </IDFactura>
+            <HuellaTBAI>
+                <EncadenamientoFacturaAnterior t-if="chain_prev_invoice">
+                    <SerieFacturaAnterior t-out="chain_prev_invoice.l10n_es_tbai_sequence"/>
+                    <NumFacturaAnterior t-out="chain_prev_invoice.l10n_es_tbai_number"/>
+                    <FechaExpedicionFacturaAnterior t-out="datetime_strftime(chain_prev_invoice.l10n_es_tbai_registration_date, '%d-%m-%Y')"/>
+                    <SignatureValueFirmaFacturaAnterior t-out="chain_prev_invoice.l10n_es_tbai_signature[:100]"/>
+                </EncadenamientoFacturaAnterior>
+                <Software>
+                    <LicenciaTBAI t-out="license.license_number"/>
+                    <EntidadDesarrolladora>
+                        <NIF t-out="license.license_nif"/>
+                    </EntidadDesarrolladora>
+                    <Nombre t-out="license.software_name"/>
+                    <Version t-out="license.software_version"/>
+                </Software>
+                <NumSerieDispositivo>TEST-DEVICE-001</NumSerieDispositivo>
+            </HuellaTBAI>
+        </template>
+
+        <template id="template_invoice_sujetos">
+            <Emisor>
+                <NIF t-out="sender_vat"/>
+                <ApellidosNombreRazonSocial t-out="sender.name"/>
+            </Emisor>
+            <Destinatarios t-if="is_emission and recipient">
+                <IDDestinatario>
+                    <NIF t-if="recipient.get('nif')" t-out="recipient['nif']"/>
+                    <IDOtro t-else="">
+                        <CodigoPais t-if="recipient.get('alt_id_country')" t-out="recipient['alt_id_country']"/>
+                        <IDType t-out="recipient['alt_id_type']"/>
+                        <ID t-out="recipient['alt_id_number']"/>
+                    </IDOtro>
+                    <t t-set="partner" t-value="recipient['partner']"/>
+                    <ApellidosNombreRazonSocial t-out="partner.name"/>
+                    <CodigoPostal t-out="partner.zip"/>
+                    <Direccion t-out="recipient['partner_address']"/>
+                </IDDestinatario>
+            </Destinatarios>
+            <VariosDestinatarios t-if="is_emission">N</VariosDestinatarios>
+            <!-- Odoo does not support multi-recipient invoices (TBAI does)-->
+            <EmitidaPorTercerosODestinatario t-if="is_emission" t-out="thirdparty_or_recipient"/>
```

Comment:
I've been testing other options lately. 
If Bizkaia works it will require the option "D", if accounting firms use this they might require option "T".
Thought we could leave it just in case so we don't have to update the template.

Reply:
It could be hard-coded back if you prefer.